### ISSUE TITLE
The exp claim is added by default

### ISSUE THREAD

**Issue by jmg1138** [claudiobernasconi/SimpleJWT#1]
I didn't expect the `exp` claim to be added by default.  

On the readme it states:  

> Note: If you want to have the exp-claim which will be verified during decoding you have to add the ExpirationClaim to the collection of IStandardClaim's you pass to the constructor.

But even if I do not add that to the constructor, it still puts the `exp` claim into my token, which causes them to expire when I don't want them to expire. I would have preferred to be able to handle my token expiration myself.  

I know this repository hasn't been updated in a while, but I wanted to note this here in case anyone else runs into this.

**Comment by claudiobernasconi** [comment]
Thanks for mentioning it. Although the library is a little old, I can still see myself fixing bugs. You are right; the documentation isn't on par with the implementation.

However, the longer I think about it, I'm not sure how to solve the issue. I could remove the exp claim from the standard claims. However, I don't want to encourage bad design, and I believe that having an expiration and validating it is required to improve security.

What if there was an option to set the duration of the exp claim? Today, it's fixed to 30 minutes. I can see that you might want to extend it. Are you sure you really have a valid use case where you don't want to verify the age of a token?

I'm interested to hear your thoughts about it. I might make a fix for it.

**Comment by jmg1138** [comment]
I agree with you that having a short expiration and having an expiration by default is best. In this case, maybe the only change I would recommend is to your README so that it indicates that the `exp` claim is included with the standard claims and is fixed at 30 minutes.

I was planning to allow a longer expiration time for my tokens but I've changed my mind on that now. I'm in the middle of completely changing how I am implementing JWT authorization and authentication, using .NET role based authorization and the System.IdentityModel.Tokens.Jwt package. In the future, if I want to extend the expiration, I will probably setup something like a refresh token.

As a side note, I did try out [JWT](https://www.nuget.org/packages/JWT), and as I mentioned am going to end up using [System.IdentityModel.Tokens.Jwt](https://www.nuget.org/packages/System.IdentityModel.Tokens.Jwt/), and neither of those automatically added an `exp` claim to the tokens.
